docs/src/ci.md

```diff
@@ -312,7 +312,7 @@ tests:
 
 By default, Playwright downloads browser binaries when the Playwright NPM package
 is installed. The NPM packages have a `postinstall` hook that downloads the browser
-binaries. This behavior can be [customized with environment variables](./library.md#managing-browser-binaries).
```

Comment:
This is upstream (playwright.dev) managed via the links. For JS its using the library.md and for the rest the installation.md.